app/controllers/admin/ticket_scannings_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class TicketScanningsController < Admin::BaseController
+    before_action :authenticate_user!
+    before_action :find_physical_ticket_by_token
```

Comment:
Yes I thought about this but where should I catch the exception. After loading physical_ticket, ticket_scanning object should be loaded and authorized so I don't know where should I catch the exception. 🤔